### DISCUSSION TITLE
Transfering GitHub account content to another account I already own?

### DISCUSSION THREAD

**Discussion by shicky4** [community/community#28023] [Other Feature Feedback, Questions, & Ideas]
As titled really, I have two GitHub accounts, one is active and one isn't, it's for the purposes of transfer. I had a support thread where they mentioned I needed to rename and swap them basically as it reserves the username for 90days.

Given that reservation period I'm concerned that will basically lock the accounts and mean I cannot transfer usernames at all.

If I delete one of them, will that username then be free'd up? So I could transfer the account to that username?

So usernameA has everything on it but is a childish username. I also have usernameB which I want to transfer usernameA content to. So I can delete usernameB then rename the usernameA account to match it?

Support is saying I should slowly swap them over but with the 90day reservation, will that not cause any problems? They haven't responded to my follow ups

Thank you

**Comment by jfullstackdev** [comment]
that's quite tricky, changing username can retire repo names, which will be problematic, 

if you can just follow this 

https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/merging-multiple-personal-accounts

then after say more than 90 days after deleting the other account, change the username of the other account, make sure you keep the email because that's linked to all the commits in the other repo, you can simply add that even unverified

**Reply by therohit777** [reply]
I think you can fork those content
